File: January 2022/Day 31/Que7.java
// https://www.codechef.com/problems/UTKPLC
// Problem Code: UTKPLC



/* package codechef; // don't place package name! */
import java.util.*;
import java.lang.*;
import java.io.*;

/* Name of the class has to be "Main" only if the class is public. */
class Codechef
{
	public static void main (String[] args) throws java.lang.Exception
	{
		// your code goes here
		Scanner scan = new Scanner(System.in);
        int T = scan.nextInt();
        while(T-- != 0){
            HashMap<Character, Integer> map = new HashMap<Character, Integer>();
            map.put(scan.next().charAt(0), 1);
            map.put(scan.next().charAt(0), 2);
            map.put(scan.next().charAt(0), 3);
            
            char x = scan.next().charAt(0);
            char y = scan.next().charAt(0);
            
            if(map.get(x)< map.get(y)) System.out.println(x);
            else System.out.println(y);
        }
	}
}

File: January 2022/Day 31/Que7.py
# https://www.codechef.com/problems/UTKPLC
# Problem Code: UTKPLC



# cook your dish here
T = int(input())
while T>0:
    c1,c2,c3 = input().split()
    
    dic = {c1:1, c2:2, c3:3}
    x,y = input().split()
    if dic[x] < dic[y]:
        print(x)
    else:
        print(y)
    T-=1